Leanplum-SDK/Classes/Features/Actions/LPActionContext.m

```diff
@@ -511,6 +512,8 @@ - (void)runActionNamed:(NSString *)name
         [Leanplum triggerAction:childContext];
     });
     LP_END_TRY
+    
+    [[LPCountAggregator sharedAggregator] incrementCount:@"run_action_named"];
```

Comment:
could you add LPCountAggregator as an instance variable / dependency to this class too?